openmetadata-ui/src/main/resources/ui/src/styles/app.less

```diff
@@ -76,6 +76,16 @@
 .text-underline {
   text-decoration: underline;
 }
+
+//  Radius
+
+.rounded {
+  border-radius: 4px;
+}
```

Comment:
You can't call 4px to rounded. Please update either class name or CSS value

Reply:
@chirag-madlani  updated the class name!